mvp-spec/json-schemas/post-seed-response.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "post-seed-response",
+    "title": "POST /v1/seed request",
```

Comment:
`POST /paf-proxy/v1/seed response` (missing prefix and incorrect "request")

Reply:
Done